### DISCUSSION TITLE
/alerts shows active alerts related with other Telegram chats

### DISCUSSION THREAD

**Discussion by cstavares** [metalmatze/alertmanager-bot#152] [Help/Support] [answered]
Hi, 

I am running two metalmatze/alertmanager-bot docker containers on same host and:

- both are listening in different ports and
- each is connected to different telegram chat 

prom/alertmanager is correctly routing events to each of those alertmanager-bot containers based on event's content but, if I type **/alerts** on any of those telegram chats alertmanager-bot replies with all active events. So, in telegram chat **_A_** I can see active events that are related with chat **_B_**.

Can alertmanager-bot query alertmanager based on tags that are known by alertmanager?
For example, alertmanager knows the name of the receivers that should contact to route the events. I am not sure if alertmanager supports this also.. Do you have any idea?

Thanks.

**Comment by metalmatze** [comment, marked as answer]
Yeah, this is something the the bots needs to pass to Alertmanager when talking to the API. However, we currently don't configure the alertmanager-bot in a way that it would know what receiver to pass to Alertmanager.
This is definitely a feature that I'm happy to merge in a PR. :slightly_smiling_face:
